packages/shared/src/utils/helpers/__tests__/durations.ts

```diff
@@ -21,14 +21,15 @@ describe('buildDurationConfig', () => {
         underlying_symbol: 'frxAUDJPY',
     };
 
-    const durations = {
+    const durations: Duration.TDurations = {
```

Comment:
For clarification, Do u need to define type here?

Reply:
@niloo-fs, this showed an error before I added the `forward: {}`. I guess I should have checked after adding that field.